src/OpenTelemetry.AutoInstrumentation/Instrumentations/MySqlData/MySqlDataCommon.cs

```diff
@@ -0,0 +1,115 @@
+// <copyright file="MySqlDataCommon.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET6_0_OR_GREATER
+
+using System.Data;
+using System.Diagnostics;
+using OpenTelemetry.Instrumentation.MySqlData;
+
+namespace OpenTelemetry.AutoInstrumentation.Instrumentations.MySqlData;
+
+internal class MySqlDataCommon
+{
+    internal const string DbName = "db.name";
+    internal const string DbSystem = "db.system";
+    internal const string DbStatement = "db.statement";
+    internal const string DbUser = "db.user";
+
+    internal const string NetPeerIp = "net.peer.ip";
+    internal const string NetPeerPort = "net.peer.port";
+    internal const string NetPeerName = "net.peer.name";
+
+    internal const string PeerService = "peer.service";
+
+    internal const string MysqlDatabaseSystemName = "mysql";
+    internal static readonly ActivitySource ActivitySource = new ActivitySource(
```

Comment:
We can update the name of the ActivitySource to `OpenTelemetry.AutoInstrumentation.MySqlData` if that would be preferred. As long as the outcome is the same (tests pass) that should be fine

Reply:
Updated in d4bcef1a1e1bfc68d3b5fba34557f423b3a88681